### ISSUE TITLE
Announce dates of staged releases

### ISSUE THREAD

**Issue by jman005** [openai/gpt-2#140]
It's great to see that OpenAI plans to release better models for public use in the future. However, I disagree with the current staged released strategy in place. Rather than announce when these releases will occur, OpenAI is just releasing them sporadically whenever the time is right for them. This is even worse than not releasing the model at all - isn't the whole point of staged releases to give time for others to prepare? Partnerships do not necessarily fix this issue; the general public should at least know when these new models will release.

**Comment by bladedsupernova** [comment]
If I knew when the next release was, I'd be able to know if I need to move my birthday over by a day.

**Comment by bladedsupernova** [comment]
So far, our frequency statistics says "every 78 days"

**Comment by WuTheFWasThat** [comment]
@jman005 yeah, I agree that it'd be useful if we had release dates; it's something we considered and which I hope we'll consider more seriously in the future!  Note though even if we did have dates, they might be provisional since we're still considering evidence as time goes so plausibly in certain scenario's we'd want to push the dates back.  Hopefully our schedule this time around won't be too erratic, either!

**Comment by dackdel** [comment]
https://twitter.com/npcollapse/status/1136559379602427904?s=21
